src/ladds/particle/BreakupWrapper.cpp

```diff
@@ -6,19 +6,28 @@
 
 #include "BreakupWrapper.h"
 
+#include <autopas/utils/ArrayMath.h>
+
 namespace LADDS {
 
 void BreakupWrapper::simulateBreakup(const CollisionFunctor::CollisionCollectionT &collisions) {
-  for (const auto &[p1, p2, dist] : collisions) {
+  for (const auto &[p1, p2, dist, position] : collisions) {
     // convert particles to satellites for breakup code
     std::vector<Satellite> satellites{
         SatelliteToParticleConverter::convertParticleToSatellite(*p1),
         SatelliteToParticleConverter::convertParticleToSatellite(*p2),
     };
+
+    auto parentID = p1->getID();
```

Comment:
```suggestion
    const auto parentID = p1->getID();
```
You are not changing it, right?